install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -0,0 +1,188 @@
+[[configuring-etcd-monitoring]]
+= Configuring etcd monitoring
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+If the `etcd` service does not run correctly, successful operation of the whole OpenShift cluster is in danger. Therefore, it is reasonable to configure monitoring of `etcd`.
+
+Follow these steps to configure `etcd` monitoring:
+
+. Verify that the monitoring stack is running:
++
+[subs="quotes"]
```

Comment:
This is different. We usually use the [config,yaml] notification, and we don't enclose the commands in asterisks.